### DISCUSSION TITLE
Who is working on what for project?

### DISCUSSION THREAD

**Discussion by matty10s** [matty10s/group3-project#4] [General]
Wireframe – Taylor Bumgarner
 
 
Homepage – Mock up – Matt Kopay
 
Home Page -  Shannon 
 
Plot Points Page – Taylor
 
Cast Page -  Matt 
 
Trivia Page -  Shannon
 
 
Mobile Response – Taylor
 
Semantic HTML – Matt
 
Flexbox – Shannon  
 
BEM formatting - Shannon
 
 
Transition and Animation – Towards End   Work on together.

**Comment by tay-baum** [comment]
[Group Project Wireframes.pdf](https://github.com/matty10s/group3-project/files/9126253/Group.Project.Wireframes.pdf)

Hi guys! Here's the wireframes I drafted up to get us a start on page layouts. Totally ok with changing any of this to fit content as needed - I'm not sure how many cast we'll want to list or how much trivia and that could definitely change layout. I tried to keep in mind what we've been doing with grids and how it would translate to mobile - I'm also not sure if we can do a hamburger menu for the mobile responsive version?? I know that's something I would have to learn. If not, we can just drop that nav bar down below the logo in the header for mobile versions.

Hopefully it can at least serve as a good reference to get us started!

**Reply by matty10s** [reply]
Thank you Taylor,  I am using this to build the mock up.  👍   Posting some files out there for review

**Comment by shannonkuhn92** [comment]
I thought the mockup was a great start! I created a Pull request for some work I did before our group meeting! I added three more .html pages (cast, episodes, trivia) and dropped those into the header links so we can go in and create the html content for our assigned page and see it render live. I copied the code from the index.html to maintain the header/footer info and just cleared out the <main> area. I also updated the stylesheet slightly to clean up the look a bit!

--MATT: Please approve the pull request so this is available for all of us to work on over the weekend :)

**Comment by matty10s** [comment]
You got it, I was gonna start working on the other pages this weekend.
I'm at a family Reunion this afternoon.

Pull request approved.   Are we meeting today July 23?  Just curious, no worries if cant.
________________________________
